ServerApp/app/__init__.py

```diff
@@ -6,41 +6,65 @@
 import subprocess
 import time
 import re
+import json
+import pickle
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider possible security implications associated with pickle module.](https://app.codacy.com/manual/rejahrehim/SecureTea-Project/pullRequest?prid=5573783)

Reply:
Handled for security